pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -265,6 +266,9 @@ def setUpClass(cls):
             cls.sync_repo,
         )
         cls.task_bodies = tuple(poll_spawned_tasks(cls.cfg, report))
+        cls.repos_iter = tuple((
+            get_repo(cls.cfg, attrs['_href']) for attrs in cls.attrs_iter
+        ))
```

Comment:
> That said, now I really don't understand what self.attrs_iter is supposed to be. The name is not descriptive, and your example seems to have assigned completely different data to it than before.

`attrs_iter` is an iterable of attributes. It can be an iterable of repository attributes, or an iterable of user attributes, or anything else really.